### PR TITLE
EZP-21109: Changed default values for min and max to null

### PR DESCRIPTION
**Pull request by jeromegamez**
> Fixes https://jira.ez.no/browse/EZP-21109
## tl;dr;

The previous defaults `0` and `false` were converted to the integers `0` and `0`, resulting in `0` being the only valid value for the given integer field.
## Details

When creating a new Integer field without defining a `minIntegerValue` and/or `maxIntegerValue` validator constraint, the default values are used, until this PR `0` (min) and `false` (max). When those values end up in the database, they are converted to integer values, resulting in `0` (min) and `0` (max).

With this configuration, the only valid value to enter in such an integer field would be `0`.

The default behavior when creating the same field in the admin interface is to keep the both values unset by default, allowing every integer value to be valid.

This change ensures this behavior.

Cheers
:octocat: Jérôme

### CODE REVIEW

**Comment by gggeek** [comment]
I am not sure if this is the cause, but there seems to be a regression (happening on 5.4), where
- if you have a content type which had set no min/max values
- you update its definition via API, without changing *anything*
- then both min and max get set to 0

**Comment by joaoinacio** [comment]
@gggeek: not certain either, but at first sight it doesn't "look" like this is it.
In any case feel free to refer to https://jira.ez.no/browse/EZP-26382
